### PR TITLE
Add Chinese translations per 南山少帅Felix in chat

### PR DESCRIPTION
**Pull request by zyphlar**

### CODE REVIEW

File: android/app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -704,6 +704,10 @@
 	<string name="enable_keep_screen_on">保持螢幕開啟</string>
 	<!-- Description in preferences -->
 	<string name="enable_keep_screen_on_description">啟用後，顯示地圖時畫面將始終開啟。</string>
+	<!-- A preference title; keep short! -->
+	<string name="enable_show_on_lock_screen">在鎖定螢幕上顯示</string>
```
Comment (biodranik) [inline android/app/src/main/res/values-zh-rTW/strings.xml:708]: @zyphlar it should be in data/strings/strings.txt

Comment (zyphlar) [reply android/app/src/main/res/values-zh-rTW/strings.xml:708]: Hmm somehow Github search didn't show this string as existing in strings.txt. I'll add.

Comment (zyphlar) [reply android/app/src/main/res/values-zh-rTW/strings.xml:708]: @biodranik is this better?

**Review by biodranik — APPROVED** [review]
Yes. We usually split strings regeneration commits from strings.txt changes to avoid merge conflicts and for easier reverting.

Thanks for taking and fixing this issue from the chat!

**Pull request merged**